.github/workflows/deploy-to-eks.yaml

```
@@ -11,15 +16,33 @@ on:
 jobs:
   deploy-ckf-to-eks:
     runs-on: ubuntu-22.04
+    env:
+      UATS_BRANCH: ${{ inputs.uats_branch }}
```

Since we 're using a UATS_BRANCH env variable, we could omit the `default` from the input and implement it here. This way, it will be closer to what we do for AKS. e.g.
```suggestion
      UATS_BRANCH: ${{ inputs.uats_branch || "main" }}
```